public/index.ts

```diff
@@ -0,0 +1,11 @@
+import charles from "./charles.jpeg";
+import full from "./fullSendHd.jpg";
+import david from "./davidIcon.jpeg"
+import mma from "./mma.png";
+import ted from "./ted.jpeg";
+import rogan from "public/rogan.jpg";
+import rich from "public/rich.jpeg";
+import lex from "public/lex.jpeg";
+import logo from "public/FClogo.jpg"
```

Comment:
This needs to be clean or removed no need to import images like this 
I already mentioned this